Source/WebCore/svg/SVGPathElement.cpp

```diff
@@ -183,6 +183,11 @@ ExceptionOr<Ref<SVGPoint>> SVGPathElement::getPointAtLength(float distance) cons
     // Spec: Clamp distance to [0, length].
     distance = clampTo<float>(distance, 0, getTotalLength());
 
+    auto* renderer = this->renderer();
+    // Spec: If current element is a non-rendered element, throw an InvalidStateError.
+    if (!renderer)
```

Comment:
No need for the local variable since it is only used in this if-statement:
```
// Spec: If current element is a non-rendered element, throw an InvalidStateError.
if (!renderer())
```

Reply:
I will update it for `SVGGeometryElement` as well. :-)

Reply:
Actually - in `SVGGeometryElement`, it is used multiple times, so I will leave it as is.

Reply:
Done in [237f550](https://github.com/WebKit/WebKit/commit/237f550c0ea8d94136d85245cda340cbd0bd720d).